File: ProjectFindDatabase/Models/Companies.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProjectFindDatabase.Models
{
    public class Companies
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string VATNumber { get; set; }
        public string Number { get; set; }
        public string Code { get; set; }
        public string Image { get; set; }
    }
}

File: ProjectFindWEB/ProjectFind/src/app/models/companies.ts
export class Companies {
	id:string;
	name: string;
	address:string;
	vatNumber: string;
	number: string;
	code: string;
	image: string;
	constructor(
		id?:string,
		name?: string,
		address?:string,
		vatNumber?: string,
		number?: string,
		code?: string,
		image?: string
	){
		this.id=id;
		this.name=name;
		this.address=address;
		this.vatNumber=vatNumber;
		this.number=number;
		this.code=code;
		this.image=image;
	}
}
